R/fd_inventory.R

```diff
@@ -3,49 +3,33 @@
 
 #' Raw inventory table.
 #'
-#' @details The columns are as follows:
-#' - `Site` (character): Replicate code. Each replicate contains
-#'   multiple plots with different disturbance treatments.
-#' - `SubplotID` (character): Subplot ID number. These subplot codes are a
-#' concatenation of the plot (\code{\link{fd_plots}}) and
-#' subplot \code{\link{fd_subplots}} codes.
-#' - `Tag` (integer): Tree tag number.
-#' - `Species` (character): Species code from the USDA Plants Database; see
-#' \url{https://plants.sc.egov.usda.gov/java/}.
-#' - `DBH_cm` (numeric): Diameter at breast height (1.37m), cm.
-#' - `Health_status` (character): live (L), moribund (M), or dead (D).
-#' - `Canopy_status` (character): Canopy status: overstory dominant (OD),
-#' overstory submissive (OS), sapling (SA), understory (UN).
-#' - `Date` (date): Inventory entry date.
-#' - `Notes` (character): Notes.
-#' - `Replicate` (character): Replicate code, extracted from `SubplotID`.
-#' - `Plot` (integer): Plot ID number, extracted from `SubplotID`.
-#' - `Subplot` (character): Subplot code, extracted from `SubplotID`.
-#' @return A `data.frame` or `tibble`. See "Details" for column descriptions.
+#' @return A `data.frame` or `tibble`. Call \code{\link{fd_metadata}} for field metadata.#'
```

Comment:
```suggestion
#' @return A `data.frame` or `tibble`. Call [fd_metadata()] for field metadata.
```

With Roxygen Markdown enabled, the above simpler form will work.